devsetup/scripts/gen-edpm-baremetal-kustomize.sh

```diff
@@ -59,39 +59,42 @@ patches:
     kind: OpenStackDataPlaneNodeSet
   patch: |-
     - op: add
-      path: /spec/services/0
-      value: repo-setup
-    - op: replace
       path: /spec/baremetalSetTemplate/bmhNamespace
-      value: ${NAMESPACE}
-    - op: replace
-      path: /spec/baremetalSetTemplate/ctlplaneInterface
-      value: enp1s0
-    - op: add
-      path: /spec/baremetalSetTemplate/provisioningInterface
-      value: ${PROVISIONING_INTERFACE}
+      value: ${EDPM_BMH_NAMESPACE}
     - op: add
-      path: /spec/env/0
-      value: {"name": "ANSIBLE_CALLBACKS_ENABLED", "value": "profile_tasks"}
+      path: /spec/nodeTemplate/networks
```

Comment:
We don't need this and can drop from both the scripts.

Reply:
Done